src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/OverflowDbEdge.java

```diff
@@ -34,12 +34,21 @@
   private final String label;
   private final VertexRef<OverflowDbNode> outVertex;
   private final VertexRef<OverflowDbNode> inVertex;
-  private int outBlockOffset = UNITIALIZED_BLOCK_OFFSET;
-  private int inBlockOffset = UNITIALIZED_BLOCK_OFFSET;
+
+  /* When storing the inVertex in the outVertex' adjacent node array, there may be multiple edges
+   * with the same (direction, label), i.e. they are stored in the same block. To be able to
+   * identify this edge, we store it's offset into that block */
+  private int outBlockOffset = UNINITIALIZED_BLOCK_OFFSET;
+
+  /** When storing the outVertex in the inVertex' adjacent node array, there may be multiple edges
+   * with the same (direction, label), i.e. they are stored in the same block. To be able to
+   * identify this edge, we store it's offset into that block */
+  private int inBlockOffset = UNINITIALIZED_BLOCK_OFFSET;
+
   private final Set<String> specificKeys;
   private boolean removed = false;
 
-  private static final int UNITIALIZED_BLOCK_OFFSET = -1;
+  public static final int UNINITIALIZED_BLOCK_OFFSET = -1;
```

Comment:
Please leave this private, it is an implementation detail of the edge.

Reply:
done